### PR TITLE
always show source for methods

### PR DESCRIPTION
**Pull request by devoncarew**
- fixed a minor UI item where field properties would should as `read-only , inherited` rather than `read-only, inherited`
- always show source code; remove the disclosure toggle; use the section h2 instead of an h4
- strip the common indent off the displayed source

### CODE REVIEW

File: lib/src/model.dart [hunk 1]
```diff
@@ -1119,7 +1119,15 @@ abstract class SourceCodeMixin {
...
       }
     }
-    return contents.substring(node.offset - (node.offset - i), node.end);
+
+    // Trim the common indent from the source snippet.
+    String source =
+        contents.substring(node.offset - (node.offset - i), node.end);
+    String remainer = source.trimLeft();
+    String indent = source.substring(0, source.length - remainer.length);
+    return source.split('\n').map((line) {
```
Comment (sethladd) [inline lib/src/model.dart]: will this work on windows?

**Comment by sethladd** [comment]
could you rebase this before we merge?

`git rebase -i master`

and then:

`git push --force`

File: lib/src/model.dart (hunk 1 above)
Comment (devoncarew) [inline lib/src/model.dart]: Yup, it'll split `\r\n` lines into things that end in `\r`, and then we'll add the `\n` back on with the join(). We could also right-trim the strings...

**Comment by devoncarew** [comment]
Will do re: rebasing

**Comment by sethladd** [comment]
Thanks!

**Pull request merged**